test/integration/admin/api/settings_controller_test.rb

```diff
@@ -32,19 +32,26 @@ def setup
   end
 
   test 'update' do
-    params = { access_token: token, signups_enabled: false, change_account_plan_permission: 'invalid' }
+    params = { access_token: token, signups_enabled: false, change_account_plan_permission: 'invalid', change_service_plan_permission: 'invalid'}
     assert 'request', settings.change_account_plan_permission
```

Comment:
what is this assertion supposed to do? `assert_equal`?

Reply:
apparently :sweat_smile: